conf/docker.yaml.template

```diff
@@ -0,0 +1,13 @@
+DOCKER:
+  # Name of the libvirt image containing the OS with docker instance set up
+  DOCKER_IMAGE: rhel7-docker-base
```

Comment:
@tpapaioa @mshriver This image is concerning in new pipeline as the image is being used from static libvirt setup. The setting and code needs to be shifted to suite new pipeline before we execute this in new pipeline.

Either in this Pr or next !

Reply:
This setting is going to be removed in the pytest conversion PR, which re-factors the tests to use broker.